automation/p4/tofino/variables.tf

```diff
@@ -35,7 +35,7 @@ variable "tofino" {
     bfrt_ami = "ami-01a5ff54de23b6739"
   }
 }
-variable "hcp_ami" {default ="ami-0a275622fd8af5f67"}
+variable "hcp_ami" {default ="ami-0479d5b8fff8e060f"}
```

Comment:
Can you update this image to have the TPS Python dependencies pre-installed for speed?